src/support/utils.js

```diff
@@ -63,6 +63,19 @@ export const sendExperimentationCandidatesMessage = async (
 });
 /* c8 ignore end */
 
+export const sentRunScraperMessage = async (
+  sqs,
+  queueUrl,
+  jobId,
+  urls,
+  slackContext,
+) => sqs.sendMessage(queueUrl, {
+  processingType: 'import',
```

Comment:
the processing type should not be import - this is reserved for Importer-as-a-service. rather you should leverage the default scraper

Reply:
`processingType` should be `scrape`?

Reply:
no, `default`

Reply:
should i create a new handler for scraping (scrape-handler.js) in spacecat-content-scraper because we need to store the scraped content to another location? also we needed meta tags to be in the scraped result, or change the base implementation altogether (abstract-handler.js)?

currently the location is `scrapes/${this.config.jobId}/scrape.json` and we want to store at `scrapes/{siteId}/.../{page_path}.json`

Reply:
the default scrape would benefit from your proposed change to store according to the page path. please do not introduce a new handler to the content scraper but instead:
- modify default scraping store location to reflect page path
- you don't need to change anything regarding job ID - simply set the job ID to the site ID when messaging the scraper
- modify the default.js executed in the puppeteer browser to return the entire DOM instead of only the body. other use-cases need that as well.

Reply:
For the page path, I think I would need to override the `processUrl` function, I have raised a PR here, will move it to default handler : https://github.com/adobe/spacecat-content-scraper/pull/137